sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.cpp

```diff
@@ -372,8 +377,7 @@ Expected<std::string> jit_compiler::calculateHash(
     // unique for each query, so drop it:
     CommandLine.pop_back();
 
-    // The command line contains the DPCPP root and clang major version in
-    // "-resource-dir=<...>" argument.
+    // TODO: Include hash of the current libsycl-jit.so/.dll somehow...
```

Comment:
Probably not that important and can be skipped for now, because different releases (and most of the commits) will likely result in different `SourceHash` due to changes in the headers.